File: messages/src/main/java/com/thundashop/core/bookingengine/data/BookingEngineConfiguration.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.thundashop.core.bookingengine.data;

import com.thundashop.core.common.DataCommon;
import com.thundashop.core.pmsmanager.TimeRepeaterData;
import java.util.LinkedHashMap;

/**
 *
 * @author ktonder
 */
public class BookingEngineConfiguration extends DataCommon {
    public boolean confirmationRequired = false;
    public RegistrationRules rules = new RegistrationRules();
    public TimeRepeaterData openingHours = null;
    public LinkedHashMap<String, TimeRepeaterData> openingHoursData = new LinkedHashMap();    
}

File: html5apps/terminal/events/core/bookingengine/data/BookingEngineConfiguration.php
<?php
class core_bookingengine_data_BookingEngineConfiguration extends core_common_DataCommon  {
	/** @var String */
	public $confirmationRequired;

	/** @var core_bookingengine_data_RegistrationRules */
	public $rules;

	/** @var core_pmsmanager_TimeRepeaterData */
	public $openingHours;

	/** @var java.util.LinkedHashMap */
	public $openingHoursData;

}
?>